doc/book/service-manager.md

```diff
@@ -0,0 +1,99 @@
+# Service Manager
+
+The `zend-log` package provides several components which can be used in 
+combination with the service manager. These components make it possible
+to quickly setup a logger instance or to provide custom writers, filters,
+formatters or processors.
+
+## LoggerAbstractServiceFactory
+
+When you register the abstract factory called `Zend\Log\LoggerAbstractServiceFactory`,
+you will be able to setup loggers via the configuration. The abstract factory can be
+registered in the service manager like this:
+
+```php
+// module.config.php
+return [
+    'service_manager' => [
+        'abstract_factories' => [
+            'Zend\Log\LoggerAbstractServiceFactory',
+        ],
+    ],
+];
+```
```

Comment:
As `Zend\Log` is now a module that provide ConfigProvider, and can be registered if using `ZendSkeletonApplication`, I think we can add note:

If you are using ZendSkeletonApplication, the `LoggerAbstractServiceFactory` will automatically registered by registering `Zend\Log` into `config/application.config.php`:

```php
return array(
    'modules' => array(
        'Zend\Log',
     ),
    //  .... 
);
```